dds/DCPS/Serializer.cpp

```diff
@@ -474,7 +474,7 @@ Serializer::read_string(ACE_CDR::WChar*& dest,
   //
   // Extract the string size.
   //
-  ACE_CDR::ULong bytecount; // includes the null
+  ACE_CDR::ULong bytecount; // Not include the null
```

Comment:
```suggestion
  ACE_CDR::ULong bytecount;
```

I would just remove it entirely, since if there's no null it you don't have to say that it's not included in the size.

Reply:
Okay. I'll remove it. Just wanted to fix the previous comment that may cause confusion.